pkg/lib/machine-dialogs.js

```diff
@@ -147,6 +149,7 @@
                 conn_options['temp-session'] = false; /* Compatiblity option */
                 conn_options['session'] = 'shared';
                 conn_options['host-key'] = machine.host_key;
+                conn_options['unknown-host'] = "1";
```

Comment:
Why is this necessary? Apparently we do have a host key? (is `!on_disk` is being used for machines other than the local host?)